lightning/src/ln/channelmanager.rs

```diff
@@ -1654,10 +1654,9 @@ where
 ///
 /// ```
 /// # use lightning::events::{Event, EventsProvider, PaymentPurpose};
-/// # use lightning::ln::channelmanager::AChannelManager;
-/// # use lightning::offers::parse::Bolt12SemanticError;
+/// # use lightning::ln::channelmanager::{AChannelManager, Bolt12CreationError};
 /// #
-/// # fn example<T: AChannelManager>(channel_manager: T) -> Result<(), Bolt12SemanticError> {
+/// # fn example<T: AChannelManager>(channel_manager: T) -> Result<(), Bolt12CreationError> {
```

Comment:
Please do this in a fixup commit. We require tests to pass on each commit after all fixups are squashed.